src/components/Main.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+
+export default function Main() {
+    return (
+        <main>
+            <div className="info--container">
+                <h4 className="info--title">About</h4>
```

Comment:
Pytanie - dlaczego to jest h4? Nagłówki musimy używać zgodnie z hierarchią dokumentu. Tutaj to bardziej wygląda na h1. Strzelam, że sugerowałeś się rozmiarem czcionki, co jest dość powszechnym błędem, także luz, ale trzeba to poprawić. 

Generalnie H1 to tytuł sekcji albo całej strony i później jeżeli w danej sekcji jest podsekcja, to ona jest h2 itd. Z książek możesz kojarzyć

1. Tytuł (h1)
1.1 Tytuł poddziału (h2)
1.1.2 Tytuł podziału poddziału (h3)